### PR TITLE
added editor config so we can all play nicely :)

### PR DESCRIPTION
**Pull request by newshorts**
Feel free to adjust these settings, it's just something for us ide users so our editors will sync up and we can avoid pesky spacing diffs

### CODE REVIEW

**Comment by BigMcLargeHuge** [comment]
Hah, nicely done sir!

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+insert_final_newline = true
+
+[*.md]
+indent_style = space
+indent_size = 2
+
+# ignore third party
+[node_modules/**.js]
+indent_size = N/A
+indent_style = N/A
```
Comment (tomlutzenberger) [inline .editorconfig]: You missed a final newline as defined in line 12 😉

Comment (newshorts) [inline .editorconfig]: Woah...you're actually reviewing these PRs! 😅

Comment (tomlutzenberger) [inline .editorconfig]: Of course. Details == Quality

**Pull request merged**